UsbAccess/UsbAccess.cpp

```diff
@@ -508,6 +563,172 @@ namespace Plugin {
         returnResponse(result);
     }
 
+    uint32_t UsbAccess::getPartitionCountWrapper(const JsonObject &parameters, JsonObject &response)
+    {
+        LOGINFOMETHOD();
+        bool result = true;
+        char buffer[32] = "";
+        string command =  "lsblk -o NAME,MOUNTPOINT | grep \"sd[a-z]\" | grep \"/\" |wc -l";
+        FILE* fp = popen(command.c_str(), "r");
+        if (fp == NULL) 
+        {
+            result = false;
+        }   
+        
+        while (fgets(buffer, 32, fp) != NULL);
```

Comment:
In case fp is NULL, this might cause issues. When fp is NULL just return from the function with proper error.

Reply:
As per review comment, reused getMounted to return the list size so there was no need of fp.

  ```
  uint32_t UsbAccess::getPartitionCountWrapper(const JsonObject &parameters, JsonObject &response)
    {
        LOGINFOMETHOD();
        bool result = true;
        std::list<string> paths;
        result =  getMounted(paths);
        int PartitionCount = paths.size();
        response["PartitionCount"] = PartitionCount;
        returnResponse(result);
    }
```